### PR TITLE
Add ICPC logo to homepage

### PR DESCRIPTION
**Pull request by Peterchauu**
Fix 1115:

- Added 'ICPC' team to the homepage
- Sorted all of the teams alphabetically
- Added an additional third row of team members to compensate for new teams

feat-1115

### CODE REVIEW

**Comment by Peterchauu** [comment]
- Shortened ICPC Title

image

File: src/routes/(site)/teams.svelte [hunk 1]
```diff
@@ -66,6 +68,20 @@
           >
         </div>
       </div>
+      <!--ICPC -->
+      <div class="hero-inner-container">
+        <a href="/teams#icpc"><img src="/assets/icpc-logo.svg" alt="ICPC Team Badge" /></a>
+        <div class="size-md acm-heavy hero-text">
+          <span
+            >Ready for a challenge? Scratch your competitive itch at the <span class="acm-orange"
```
Comment (26samaahmed) [inline src/routes/(site)/teams.svelte]:
```suggestion
            >Ready for a challenge? Scratch your competitive itch with the <span class="acm-orange"
```

File: src/routes/(site)/teams.svelte [hunk 2]
```diff
@@ -66,6 +68,20 @@
...
+              >ICPC</span
```
Comment (26samaahmed) [inline src/routes/(site)/teams.svelte:77]:
```suggestion
              >ICPC Team</span
```

**Review by 26samaahmed — CHANGES_REQUESTED** [review]
All looks good. I think we should re-word the description a bit.

**Comment by Peterchauu** [comment]
- Changed the description of the ICPC team

image

**Comment by 26samaahmed** [comment]
Perfect! Everything looks good for now, we will work on adding the Node buds logo once the Design Team Finalizes it! Nice job @Peterchauu 👏

**Review by 26samaahmed — APPROVED** [review]
Perfect! Everything looks good for now, we will work on adding the Node buds logo once the Design Team Finalizes it! Nice job @Peterchauu 👏

**Pull request merged**